### PR TITLE
Prepare release 0.30.0

### PR DESCRIPTION
**Pull request by jpkrohling**
Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

## Short description of the changes
- Release preparation for v0.30.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,24 @@
 Changes by Version
 ==================
 
-0.29.1 (unreleased)
+0.30.1 (unreleased)
```
[the remark this answers is not in the document]
Comment (jpkrohling) [reply CHANGELOG.md]: Ok, I was just following the previous release :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,12 +1,24 @@
...
 -------------------
 
 - Nothing yet
 
-0.29.0 (2018-06-7)
+0.30.0 (2018-07-04)
```
[the remark this answers is not in the document]
Comment (jpkrohling) [reply CHANGELOG.md]: I can make a summary, sure.

[the remark this answers is not in the document]
Comment (jpkrohling) [reply CHANGELOG.md]: Summary made. How does it look?

**Comment by yurishkuro** [comment]
Looks like we have a bug in the release process as the build fails trying to access encrypted variables which are not accessible from a fork. Do you want to try to fix it, or merge?

**Comment by jpkrohling** [comment]
> Do you want to try to fix it, or merge?

I'll merge it. I'm afraid the problem is bigger than what it initially appears.

**Comment by jpkrohling** [comment]
Except that I can't merge it...  @yurishkuro would you do the honors ?

image

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,12 +1,41 @@
...
+1. Related to the point above: we now have also a SPI package, intended to be consumed by service providers (components
+implementing `Sender`s, `Reporter`s, `Sampler`s). If you are implementing a service, do let us know! We need your
+feedback before moving forward with this API.
+
+1. Jaeger types related to the OpenTracing standard were renamed, to avoid name clashes: `io.jaegertracing.Tracer`
+is now `JaegerTracer`.
+
+Complete list of changes:
+
+- Define some classes internal (#470, [@jpkrohling](https://github.com/jpkrohling))
```
Comment (pavolloffay) [inline CHANGELOG.md]: The links to PRs are not clickable in the changelog

Comment (jpkrohling) [reply CHANGELOG.md]: Oops. https://github.com/jaegertracing/jaeger-client-java/pull/476

**Pull request merged**